src/components/sanitized-content.tsx

```
@@ -0,0 +1,23 @@
+
+  useEffect(() => {
+    const loadDOMPurify = async () => {
+      if (typeof window !== 'undefined') {
+        const DOMPurify = (await import('isomorphic-dompurify')).default;
+        setSanitizedHtml(DOMPurify.sanitize(html));
+      }
+    };
+    loadDOMPurify();
+  }, [html]);
+
+  return <div dangerouslySetInnerHTML={{ __html: sanitizedHtml }} />;
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)